transport/http/client_test.go

```diff
@@ -99,7 +99,7 @@ func TestHTTPClient(t *testing.T) {
 
 func TestHTTPClientBufferedStream(t *testing.T) {
 	var (
-		testbody = "testbody"
+		testbody = string(make([]byte, 6000))
```

Comment:
Perhaps a comment explaining the choice of 6000? Or, better yet, factor it out into a const, e.g.

```go
var (
    bodysize = 6000 // trigger the foo in the bar package
    testbody = bytes.Repeat('a', bodysize)
)
```

Reply:
I has not an specific meaning by itself, it big enough to make the test fail hehe. If it where to be over 9000 then we would have to modify the test as the `_, err = response.Body.Read(b)` would not return `io.EOF` but on the second read would do it (I explained this on the main issue hehe).

I added the documentation too :+1: